encoders/image/VSEImageEncoder/requirements.txt

```diff
@@ -1,3 +1,3 @@
 jina
```

Comment:
Should `jina` be here?

Reply:
Yes, because the image inside the Docker is not based on jina image